day_7/10_little_monkeys.rb

```diff
@@ -0,0 +1,48 @@
+class MonkeyBed
+  attr_accessor :monkeys, :plural, :last_sentence
```

Comment:
I do think that this is a pretty unique solution. Nice work. However, I'd recommend trying out a simpler solution with a loop. Should only require one method definition.